cli/defenseclaw/commands/cmd_doctor.py

```
@@ -3934,22 +4964,88 @@ def _run_fixers(
+            title == "gateway token"
+            and dotenv_safety_problem
+            and not getattr(cfg, "_doctor_gateway_token_rotation_required", False)
+        ):
+            outcome = (
+                "fail",
+                f"blocked because {dotenv_safety_problem}; review and securely replace .env before credential repair",
+            )
         else:
             try:
                 outcome = fn(cfg, assume_yes=assume_yes)
             except Exception as exc:  # defensive — one fixer shouldn't abort the rest
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)